man/avocado.rst

```diff
@@ -28,62 +28,106 @@ The following list of options are builtin, application level `avocado`
 options. Most other options are implemented via plugins and will depend
 on them being loaded::
```

Comment:
Well the output looks like this:

```
SYNOPSIS
       avocado  [-h] [-v] [--config [CONFIG_FILE]] [--show [STREAM[:LVL]]] [-s] {config,diff,distro,exec-path,list,multiplex,plugins,run,sysinfo}
       ...

DESCRIPTION
       Avocado is a modern test framework that is built on the experience accumulated with autotest (http://autotest.github.io).

       avocado is also the name of its test runner command line tool, described in this man page.

       For more information about the Avocado project, please check its website: http://avocado-framework.github.io/

OPTIONS
       The following list of options are builtin, application level avocado options. Most other options are  implemented  via  plugins  and  will
       depend on them being loaded:

          -h, --help            show this help message and exit
          -v, --version         show program's version number and exit
          --config [CONFIG_FILE]
```

I think it'd be useful to repeate the `avocado -h` command at the beginning of the OPTIONS as the DESCRIPTION makes it a bit distant. Also below you use `$ avocado multiplex --help` followed by the output, which is easier to follow.